docs/UserGuide.md

```diff
@@ -80,8 +80,79 @@ hardware, os configuratoin, etc (unless available through the api) out of scope.
 
 # Example use cases
 
+To Compare a known valid reference configuration with a live cluster:
+
+`kubectl cluster-compare -r <referenceConfigurationDirecotry>`
+
+To Compare a known valid reference configuration with a local set of CRs:
+
+`kubectl cluster-compare -r <referenceConfigurationDirecotry> -f <inputConfiguration>`
+
+To Compare a known valid reference configuration with a live cluster and with a user config:
+
+`kubectl cluster-compare -r <referenceConfigurationDirecotry> -c <userConfig>`
+
+To Run a known valid reference configuration with a must-gather output:
+
+`kubectl cluster-compare -r <referenceConfigurationDirecotry> -f "must-gather*/*/cluster-scoped-resources","must-gather*/*/namespaces" -R`
+
 # Understanding the output
 
 # Options and advanced usage
 
+## Kubectl Environment Variables
+The tool is responsive to KUBECTL_EXTERNAL_DIFF environment variable (same as oc diff). This allows you to tailor the output formatting to suit your preference. 
+
+side-by-side comparison (total width 150 characters) with:
+`KUBECTL_EXTERNAL_DIFF="diff -y -W 150"`
+
 # Troubleshooting
+
+## False Positives
+
+### Reference Designs CRs may change (or not) from one cluster version to another version
+The tool sometimes reports CR to be missing completely while in the live cluster environment the CR is present. This can happen at times because the tool is not correctly identifying the operator versions of these CRs or template's critical fields don't match cluster CRs.
+
+#### Example #1:
+```
+Missing required CRs: 
+DU-Reference:
+  Tuning:
+  - ConsoleOperatorDisable.yaml
+  - DisableSnoNetworkDiag.yaml
+
+[cluster]$ kubectl get network -A
+NAME      AGE
+cluster   5d20h
+[cluster]$ kubectl get console -A
+NAME      AGE
+cluster   5d20h
+```
+#### Example #2:
+```
+Cluster CR
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: $name
+  namespace: openshift-sriov-network-operator
+  
+Template CR
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: sriov
+  namespace: openshift-sriov-network-operator
```

Comment:
Suggest to not use telco keywords like DU-Reference, Tuning for the wider audience, . You can use terms like "ExamplePart" "ExampleComponent".